common-lib/src/main/scala/com/gu/mediaservice/lib/config/CommonConfig.scala

```diff
@@ -50,6 +50,11 @@ trait CommonConfig {
 
   lazy val thrallKinesisStream = properties("thrall.kinesis.stream.name")
 
+  lazy val requestMetricsEnabled: Boolean = properties.getOrElse("metrics.request.enabled", "false").toLowerCase match {
```

Comment:
we can remove the `match` here in favour of

```suggestion
  lazy val requestMetricsEnabled: Boolean = properties.getOrElse("metrics.request.enabled", "false").toLowerCase == "true"
```